### PR TITLE
Closed Job Poster Apply Button

### PR DESCRIPTION
**Pull request by tristan-orourke**

### CODE REVIEW

**Comment by tristan-orourke** [comment]
I threw some temporary styling on the disabled button, with the class _job-post__apply-button--closed_, but @joshdrink should make it nice.

File: resources/views/applicant/job_post/apply.html.twig [hunk 1]
```diff
@@ -57,24 +57,28 @@
 
     </div>
 
-    {# Tristan, we need to check for the user's log state to display the appropriate button here. #}
-
     <div
         class="job-post__button-wrapper">
 
-        <a
+        {% if not job.isOpen() %}
```
Comment (Khristophor) [inline resources/views/applicant/job_post/apply.html.twig]: @tristan-orourke if you take a look at [my PR for previewing Job Posters](https://github.com/GCTC-NTGC/TalentCloud/pull/614/files#diff-a6fec36e1f2745cb1f4d19ec2dc4f4e2R132) I tried to move the logic for the Apply Button back into the Controller for the sake of more concise templating.

**Comment by Khristophor** [comment]
Nice work on the seeders 👍

**Comment by joshdrink** [comment]
Okay I chatted with @Khristophor about this and I think I'm going to hold off on styles until this is merged with his other code on `dev` because it directly alters the logic and how the template acts. Gimme a shout when this is ready for me.

**Comment by tristan-orourke** [comment]
I fixed the conflicts, but Josh should take one last look at styling.

**Comment by joshdrink** [comment]
The button is now grey and I altered the cursor to be an arrow instead of pointer.

**Pull request merged**